app-metrics/unbound_exporter/files/unbound_exporter.service

```
@@ -0,0 +1,17 @@
+[Unit]
+Description=Prometheus unbound exporter
+Documentation=https://github.com/kumina/unbound_exporter
+After=network-online.target
+
+[Service]
+EnvironmentFile=/etc/default/%p
+User=prometheus-exporter
```

```suggestion
User=prometheus
```
`acct-user/prometheus` offer `prometheus` user